tensorboard/BUILD

```diff
@@ -399,6 +399,14 @@ py_library(
     visibility = ["//visibility:public"],
 )
 
+py_library(
+    name = "expect_requests_installed",
```

Comment:
We should take an explicit dep on this in our `setup.py` now that it's a direct dependency, right?

Reply:
Yes—thanks!